test/verify/check-networkmanager-wireguard

```diff
@@ -127,10 +137,21 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         b.wait_not_present("#network-wireguard-settings-dialog")
         b.wait_in_text(f"#networking-interfaces th:contains('{iface_name}') + td", f"1.2.3.4/32, {m1_ip4}/24")
 
+        b.click(f"#networking-interfaces button:contains('{iface_name}')")
+        b.wait_visible("#network-interface")
+        b.click("#networking-edit-wg")
+        b.wait_visible("#network-wireguard-settings-dialog")
+        b.set_input_text("#network-wireguard-settings-keepalive-peer-0", "120")
+        psk = m1.execute("head -c 32 /dev/random | base64").strip()
+        b.set_input_text("#network-wireguard-settings-presharedkey-peer-0", psk)
```

Comment:
Please also validate these settings in the API or with `nmcli` -- the UI doesn't reflect which key mode is being used, and which keepalive time.

Reply:
Yes, good idea! Done, thanks!

Reply:
I did it with `wg` because `nmcli` doesn't show peer information.